toparea-text-widget/style/base.css

```diff
@@ -0,0 +1,13 @@
+/*
+    See the JupyterLab Developer Guide for useful CSS Patterns:
+
+    https://jupyterlab.readthedocs.io/en/stable/developer/css.html
+*/
+
+/* stylelint-disable selector-class-pattern */
```

Comment:
We need to change this rule in config (in package JSON) here and in the template as it goes against our style conventions: https://jupyterlab.readthedocs.io/en/stable/developer/css.html#css-class-naming-conventions.

Reply:
I opened https://github.com/jupyterlab/extension-template/pull/52

Reply:
```suggestion
```

This can be removed given https://github.com/jupyterlab/extension-template/pull/52, right?